global.json

```diff
@@ -1,17 +1,17 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.7.23376.3",
+    "version": "8.0.100-rc.1.23455.8",
     "allowPrerelease": true,
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "8.0.100-preview.7.23376.3",
+    "dotnet": "8.0.100-rc.1.23455.8",
```

Comment:
All the errors are:

error : Could not resolve SDK "Microsoft.NET.Sdk". Exactly one of the probing messages below indicates why we could not resolve the SDK. Investigate and resolve that message to correctly specify the SDK.

I'm guessing this is because this isn't the public RC1 SDK version, it should be rc.1.**23463.5** instead.  I tried updating this PR but I don't have permission to push to the main fork.  Alternately, we may be able to use the just released RC2 build, 8.0.100-rc.2.23502.2.  (See https://dotnet.microsoft.com/en-us/download/dotnet/8.0).

FYI @beccamc